api/v1/cluster_webhook.go

```diff
@@ -274,13 +275,15 @@ func (r *Cluster) Validate() (allErrs field.ErrorList) {
 		r.validateCerts,
 		r.validateBootstrapMethod,
 		r.validateStorageConfiguration,
+		r.validateWalStorageConfiguration,
```

Comment:
I opened https://github.com/cloudnative-pg/cloudnative-pg/issues/514 to refactor and fix the two different error messages issue over the storage size.